src/view/components/app.ts

```diff
@@ -26,7 +27,7 @@ import {MdIcon, MdIconRegistry} from '@angular2-material/icon/icon';
                 <h1>DotCMS</h1>
             </div>
             <dot-global-search></dot-global-search>
-            <div class="user">freddy@dotcms.com</div>
+            <dot-toolbar-notifications></dot-toolbar-notifications>
```

Comment:
I create this component but also separate components for notification list and notification items, thinking that we can have a notification list someplace else.